src/Core/AdminConsole/Utilities/DebuggingInstruments/UserInviteDebuggingLogger.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Text.Json;
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Microsoft.Extensions.Logging;
+using Quartz.Util;
+
+namespace Bit.Core.AdminConsole.Utilities.DebuggingInstruments;
+
+public class UserInviteDebuggingLogger(ILogger<UserInviteDebuggingLogger> logger) : IUserInviteDebuggingLogger
```

Comment:
Would it make sense to just have these be extension methods to the ILogger<T>?

Something like this 
```
public static class LoggerClass
{
    public static void LogUserStuff(this ILogger<OrganizationService> logger, IEnumerable<OrganizationUser> users)
    {
        // All your user log logic
    }
}
```
OrgService
```
    public async Task<IEnumerable<Tuple<OrganizationUser, string>>> ResendInvitesAsync(Guid organizationId, Guid? invitingUserId,
        IEnumerable<Guid> organizationUsersId)
    {
        var orgUsers = await _organizationUserRepository.GetManyAsync(organizationUsersId);
        _logger.LogUserStuff(orgUsers);

        var org = await GetOrgById(organizationId);

Reply:
I like this solution - it's lighter weight than defining a whole new service for it.